### ISSUE TITLE
Results lacklustered (specifically for "See")

### ISSUE THREAD

**Issue by xeruf** [Lusito/dict.cc-translation#56]
When searching for the term "See", the extension does not show the translations for the German noun at all.

What would also be nice is a regard for case - if I uppercase the word, it should list uppercase results first.

**Comment by Lusito** [comment]
true, but both of your requests are more of a request for dict.cc itself (I am only the author of the extension). The extension only shows what dict.cc itself returns

**Comment by xeruf** [comment]
Totally right, I wrote Paul about it. Though the plugin could simply request and combine both directions.
